src/pages/pmarket/index.ts

```diff
@@ -0,0 +1,63 @@
+import './pmarket.css';
+
+import { getTSCSpyOld } from '../../utils/api.js';
+import { waitForElement } from '../../utils/dom.js';
+import { formatSpy } from '../../utils/format.js';
+import Settings from '../../utils/local-storage.js';
+import Logger from '../../utils/logger.js';
+import Page from '../page.js';
+
+const PMARKET_MEMBERS_SELECTOR =
+  '#mainContainer > div.content-wrapper.spring > div.points-market > div.chart-main-wrap > ul:nth-of-type(2)';
+const ID_HREF_SELECTOR = 'a[href^="/profiles.php?XID="]';
+
+export const pMarket = new Page({
```

Comment:
Could you rename this to PointsMarket? (I know the naming isn't consistent at least for the time being, but I'm working on it :P)